packages/components/src/styles/ButtonGroup.module.css

```diff
@@ -7,27 +7,18 @@
 }
 
 .compact {
+	composes: buttonGroup from './base.module.css';
```

Comment:
whoa is this a general feature of CSS Modules?!

Reply:
Yup! https://github.com/css-modules/css-modules/blob/master/docs/composition.md